cryptographic_estimators/SDEstimator/SDAlgorithms/bjmm.py

```diff
@@ -169,49 +159,41 @@ def get_optimal_parameters_dict(self):
         return a
 
     def __repr__(self):
-        """
-        """
+        """ """
         rep = "BJMM estimator for " + str(self.problem)
         return rep
 
 
 class BJMMd2(SDAlgorithm):
     def __init__(self, problem: SDProblem, **kwargs):
-        """
-        Complexity estimate of BJMM algorithm in depth 2
+        """Complexity estimate of BJMM algorithm in depth 2.
 
-        The algorithm was introduced in [BJMM12]_  as an extension of [MMT11]_.
+        The algorithm was introduced in [BJMM12] as an extension of [MMT11].
 
-        expected weight distribution::
+        The expected weight distribution is as follows:
 
             +--------------------------+-------------------+-------------------+
             | <-----+ n - k - l +----->|<--+ (k + l)/2 +-->|<--+ (k + l)/2 +-->|
             |           w - 2p         |        p          |        p          |
             +--------------------------+-------------------+-------------------+
 
-        INPUT:
+        Args:
+            problem (SDProblem): SDProblem object including all necessary parameters.
```

Comment:
is this still a valid way to describe the arguments for the automatic html docs generation?

Reply:
if I understood correctly there is an automatic html doc generation extension for this format (this napoleon sphincs extension) @Dioprz ?

Reply:
It's correct. Napoleon will produce the typical reST docstring as an intermediary step for a neat display.